Backend/app/dashboard/rider-dashboard/src-read-only/API/Action/RiderPlatform/Management/Ride.hs

```diff
@@ -0,0 +1,71 @@
+getRideRideinfo merchantShortId opCity apiTokenInfo rideId = withFlowHandlerAPI' $ Domain.Action.RiderPlatform.Management.Ride.getRideRideinfo merchantShortId opCity apiTokenInfo rideId
+
+getRideInfo :: (Kernel.Types.Id.ShortId Domain.Types.Merchant.Merchant -> Kernel.Types.Beckn.Context.City -> ApiTokenInfo -> Kernel.Types.Id.Id Dashboard.Common.Ride -> Environment.FlowHandler API.Types.RiderPlatform.Management.Ride.ShareRideInfoRes)
+getRideInfo merchantShortId opCity apiTokenInfo rideId = withFlowHandlerAPI' $ Domain.Action.RiderPlatform.Management.Ride.getRideInfo merchantShortId opCity apiTokenInfo rideId
+
+getRideRideInfo :: (Kernel.Types.Id.ShortId Domain.Types.Merchant.Merchant -> Kernel.Types.Beckn.Context.City -> ApiTokenInfo -> Kernel.Types.Id.ShortId Dashboard.Common.Ride -> Environment.FlowHandler API.Types.RiderPlatform.Management.Ride.ShareRideInfoRes)
+getRideRideInfo merchantShortId opCity apiTokenInfo rideShortId = withFlowHandlerAPI' $ Domain.Action.RiderPlatform.Management.Ride.getRideRideInfo merchantShortId opCity apiTokenInfo rideShortId
+
+getRideTripRoute :: (Kernel.Types.Id.ShortId Domain.Types.Merchant.Merchant -> Kernel.Types.Beckn.Context.City -> ApiTokenInfo -> Kernel.Types.Id.Id Dashboard.Common.Ride -> Kernel.Prelude.Double -> Kernel.Prelude.Double -> Environment.FlowHandler Kernel.External.Maps.GetRoutesResp)
+getRideTripRoute merchantShortId opCity apiTokenInfo rideId lat lon = withFlowHandlerAPI' $ Domain.Action.RiderPlatform.Management.Ride.getRideTripRoute merchantShortId opCity apiTokenInfo rideId lat lon
+
+getRidePickupRoute :: (Kernel.Types.Id.ShortId Domain.Types.Merchant.Merchant -> Kernel.Types.Beckn.Context.City -> ApiTokenInfo -> Kernel.Types.Id.Id Dashboard.Common.Ride -> Kernel.Prelude.Double -> Kernel.Prelude.Double -> Environment.FlowHandler Kernel.External.Maps.GetRoutesResp)
+getRidePickupRoute merchantShortId opCity apiTokenInfo rideId lat lon = withFlowHandlerAPI' $ Domain.Action.RiderPlatform.Management.Ride.getRidePickupRoute merchantShortId opCity apiTokenInfo rideId lat lon
+
+postRideSync :: (Kernel.Types.Id.ShortId Domain.Types.Merchant.Merchant -> Kernel.Types.Beckn.Context.City -> ApiTokenInfo -> API.Types.RiderPlatform.Management.Ride.MultipleRideSyncReq -> Environment.FlowHandler Dashboard.Common.Ride.MultipleRideSyncResp)
+postRideSync merchantShortId opCity apiTokenInfo req = withFlowHandlerAPI' $ Domain.Action.RiderPlatform.Management.Ride.postRideSync merchantShortId opCity apiTokenInfo req
+
+postRideCancel :: (Kernel.Types.Id.ShortId Domain.Types.Merchant.Merchant -> Kernel.Types.Beckn.Context.City -> ApiTokenInfo -> API.Types.RiderPlatform.Management.Ride.MultipleRideCancelReq -> Environment.FlowHandler Kernel.Types.APISuccess.APISuccess)
```

Comment:
this name is confusing, can you rename like here https://github.com/nammayatri/nammayatri/blob/main/Backend/app/dashboard/CommonAPIs/spec/ProviderPlatform/Management/API/Ride.yaml#L47
Also, you can copy the same spec from provider-dashboard if it is duplicated, to make it faster